src/quotes.rs

```diff
@@ -67,6 +67,12 @@ impl YResponse {
         Ok(quotes)
     }
 
+    pub fn metadatas(&self) -> Result<YMetaData, YahooError> {
```

Comment:
I think the "s" in should be removed, naming the function just "metadata" instead.

Reply:
You are right i have just made the correction